README.md

```diff
@@ -24,17 +24,23 @@ module "newrelic_monitoring" {
 }
 ```
 
-See [variables.tf](./variables.tf) for more information on the input variables that the module accepts. For instance, the default values for an alert's duration and threshold can be overridden. The following example shows how to do so:
+See [variables.tf](./variables.tf) for more information on the input variables that the module accepts. For instance, to enable 4xx alerts on a given endpoint you can do:
 
 ```hcl
 module "newrelic_monitoring" {
   source  = "vistaprint/monitoring/newrelic"
-  version = "1.0.0"
+  version = "3.0.0"
 
   # some fields ommitted (see previous example)
-
-  alert_error_rate_5xx_duration  = 600 # seconds
```

Comment:
I wouldn't remove the example where we show how to use overrides. Instead we can have both: one for (simple) overrides and one for the more complex `status_code_alerts`.

Reply:
I believe those particular overrides do not exist anymore. That's why I removed them.

But it's true that there are others. I'll restore the example with the ones that matter.